docs/personal_data_handling.md

```diff
@@ -0,0 +1,56 @@
+# Data handling in PMM - FAQ
+
+
+The following questions are being answered related to personal and confidential data handling in PMM:
+
+1. Which type of data is transmitted?
+
+      |**Data collection source**                                       | **Data collected** |
+      | --------------------------------------------------------------- | ------------------------------------------------------
+      | DB host to PMM                                                  | Database performance metrics <br/> SQL query examples for query analytics (optional).
+      | PMM to DB Host                                                  | DSN and credentials for database access. A separate DB user is used (read-only) to access the database.
+      | DB Host to Cloud S3                                             | Database backup - optional if PMM Administrator configures it.
```

Comment:
need to mention that this is not Percona Cloud, this is S3 storage that PMM Administrator wants to use as DB backup storage. It can be any S3 in public clouds (AWS,GCP, etc) or internal S3-compatible storage